modules/SystemTest/src/server/routes/SystemTestRoutes.coffee

```diff
@@ -22,59 +24,118 @@ request = require('request')
 _ = require 'underscore'
 fs = require 'fs'
 path = require 'path'
+crypto = require 'crypto'
 
-#Create Bob User
-exports.getACASBobUser = (callback) ->
+# Get a Test user by username
+exports.getTestUser = (username, callback) ->
 	options =
 		method: 'GET'
-		url: "#{config.all.server.nodeapi.path}/api/users/bob"
+		url: "#{config.all.server.nodeapi.path}/api/users/#{username}"
 		json: true
 	request options, (error, response, body) ->
 		if error
 			throw new Error(error)
 		callback body
 
-exports.getOrCreateACASBob = (req, resp) ->
-	exports.getOrCreateACASBobInternal (statusCode, output) ->
+# Create a test user
+exports.createTestUserInternal = (username, password, callback) ->
+	# Get a base64 encoded SHA-1 hash of the password
+	sha = crypto.createHash 'sha1'
+	sha.update password
+	passwordHash = sha.digest 'base64'
+	# Format the request to create the user
+	options = 
+		method: 'POST'
+		url: "#{config.all.client.service.persistence.fullpath}authors/jsonArray"
+		json: true
+		body:
+			[
+				firstName: username
```

Comment:
Aesthetic choice: I thought it was overkill to pass in first & last names, so I'm reusing the username as much as possible for these test accounts.